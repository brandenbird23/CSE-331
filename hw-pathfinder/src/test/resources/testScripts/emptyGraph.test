# Test for when graph is empty

CreateGraph graph

ListNodes graph

FindPath graph A B