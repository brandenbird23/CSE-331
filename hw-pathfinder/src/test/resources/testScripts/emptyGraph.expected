# Test for when graph is empty

created graph graph

graph contains:

unknown: A
unknown: B